crates/pallet-domains/src/staking.rs

```diff
@@ -1604,7 +1593,7 @@ pub(crate) mod tests {
     fn test_register_operator() {
         let domain_id = DomainId::new(0);
         let operator_account = 1;
-        let operator_free_balance = 1500 * SSC;
+        let operator_free_balance = 2500 * SSC;
```

Comment:
why the change here though ?

Reply:
It needs more stake for the operator to register twice, it used to fail with a duplicate key error, but should succeed now. (If we don’t have extra stake it fails with a not enough funds error.)